crates/rspack_core/src/tree_shaking/visitor.rs

```diff
@@ -1266,24 +1266,7 @@ impl<'a> ModuleRefAnalyze<'a> {
     {
       return Some(SideEffectType::Configuration(*side_effects))
     }
-
-    let resource_data = self
-      .module_graph
-      .module_by_identifier(&self.module_identifier)
-      .and_then(|module| module.as_normal_module())
-      .map(|normal_module| normal_module.resource_resolved_data())?;
-    let resource_path = &resource_data.resource_path;
-    let description = resource_data.resource_description.as_ref()?;
-    let package_path = description.dir().as_ref();
-    let side_effects = SideEffects::from_description(description)?;
-
-    let relative_path = resource_path.relative(package_path);
-    let side_effects = Some(get_side_effects_from_package_json(
-      side_effects,
-      relative_path,
-    ));
-
-    side_effects.map(SideEffectType::Configuration)
+    None
```

Comment:
We have analyzed side-effects from package.json before, https://github.com/web-infra-dev/rspack/blob/d31aa6d30597b415da02af221883f7232c4daef9/crates/rspack_core/src/normal_module_factory.rs#L521-L540